api/grant.go

```diff
@@ -166,6 +166,6 @@ func (r GrantRequest) ValidationRules() []validate.ValidationRule {
 }
 
 type UpdateGrantsRequest struct {
-	GrantsToAdd    []GrantRequest `json:"grantsToAdd"`
-	GrantsToRemove []GrantRequest `json:"grantsToRemove"`
+	GrantsToAdd    []GrantRequest `json:"grantsToAdd" note:"Array of grant objects. See POST api/grants for more"`
```

Comment:
Might be better to use `List of` rather than `Array of`. `List` sounds more natural